tests/System/client_proxy.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# This is a generic list of commands run from a client for testing proxy manager client.
+#
+# Submitter should follow through the logs
```

Comment:
There's also something similar in https://github.com/DIRACGrid/DIRAC/blob/rel-v6r22/tests/System/client_core.sh so in case you have more tests just modify that file, which is what is run in the DIRAC certification.

Reply:
done